### PR TITLE
Case Study 1

### PR DESCRIPTION
**Pull request by brianandreasen1983**
Validated all html and validated the css file. The activities link does work currently as I was already working within Chapter 5. I have removed the CSS for the start of the next case study portion and only included the CSS for this assignment.

### CODE REVIEW

**Comment by brianandreasen1983** [comment]
Forgot to add the margin-bottom to css. I didn't see the last task in the chapter 4 case study until now my mistake.

**Comment by mbMosman** [comment]
Hey!  Make sure that you don't rename the repository, that messes with my grading stuff.  If you would be so kind as to rename that back to ptCaseStudy, grading would go much smoother.

**Comment by mbMosman** [comment]
Also, all of the images and multimedia files were already provided for you in the images and media folders, you shouldn't add them again.  Please delete the extra files and update your HTML & CSS as needed to use the files in the directories.

File: yurts.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+<nav>
+    <a href="index.html"><b>Home</b></a>&nbsp;&nbsp;
+    <a href="yurts.html"><b>Yurts</b></a>&nbsp;&nbsp;
+    <a href="activities.html"><b>Activities</b></a>&nbsp;&nbsp;
+    <a href="Reservations.html"><b>Reservations</b></a>&nbsp;&nbsp;
+</nav>
+
+<main>
+
+    <h2>The Yurts at the Pacific Trails</h2>
```
Comment (mbMosman) [inline yurts.html]: "the" is extra and not needed here.

File: yurts.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    <dl>
+        <dt><strong>What is a Yurt?</strong></dt>
```
Comment (mbMosman) [inline yurts.html]: Typos in content will count against you so be careful with this. Imagine this is a site for a business owner. They'll care if things are spelled incorrectly, capitalization is off, and/or punctuation is bad.

Yurts should not be capitalized in the above line.

File: yurts.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+        <dd>Our luxury yurts are permanent structures four feet off the ground. Each yurt has canvas walls, a wooden floor, and a roof dome that can be opened.</dd>
+
+        <dt><strong>How are the yurts furnished?</strong></dt>
+
+        <dd>Each yurt is furnished with a queen-size bed with down quilt and gas-fired stove.The luxury camping experience also includes electricity and a sink with hot and cold running water. Shower and restroom facilities are located in the lodge.</dd>
```
Comment (mbMosman) [inline yurts.html]: Space missing after the .

File: yurts.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    <title>Pacific Trails Resort::Yurts</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href=pacific.css>
+</head>
+
+<header>
+    <h1>Pacific Trails Resort </h1>
+</header>
+
+<nav>
```
Comment (mbMosman) [inline yurts.html]: The nav links should have been bold, but I'm guessing you did that w/ the CSS and pulled it out.

**Comment by mbMosman** [comment]
Overall, nice work.  A few small things to fix, but good job.

**Pull request closed without merging**